src/plugins/data/common/index_patterns/types.ts

```diff
@@ -185,6 +185,8 @@ export interface FieldSpec {
   readFromDocValues?: boolean;
   subType?: IFieldSubType;
   indexed?: boolean;
+  autoCompleteValues?: string[];
```

Comment:
Could we use the data views field spec instead ? That way we know we aren't messing with non experimental stuff